File: apm-method-plugin/src/main/groovy/com/will/method/plugin/MethodExtension.java
package com.will.method.plugin;

import com.will.base.RunVariant;

public class MethodExtension {

    public RunVariant runVariant = RunVariant.ALWAYS;
    public boolean duplcatedClassSafeMode = false;

    @Override
    public String toString() {
        return "MethodExtension{" +
                "runVariant=" + runVariant +
                ", duplcatedClassSafeMode=" + duplcatedClassSafeMode +
                '}';
    }
}

File: apm-method-plugin/src/main/kotlin/com/will/method/plugin/MethodExtension.kt
package com.will.method.plugin

import com.will.base.RunVariant

class MethodExtension {

    var runVariant = RunVariant.ALWAYS
    var duplcatedClassSafeMode = false

    override fun toString(): String {
        return "MethodExtension{" +
                "runVariant=" + runVariant +
                ", duplcatedClassSafeMode=" + duplcatedClassSafeMode +
                '}'.toString()
    }
}
